lib/validator/validator_component.cpp

```diff
@@ -12,14 +13,16 @@ namespace Validator {
 using namespace AST::Component;
 
 Expect<void> Validator::validate(const AST::Component::Component &Comp) {
-  spdlog::warn("component validation is not done yet."sv);
+  spdlog::info("Started Component Model Validation."sv);
```

Comment:
Do we finish all of the component model validation? If not, we should still keep the warning message until everything is done. But it's fine to modify it into `Component Model Validation is in active development` or some other message to describe the current status.